common/djangoapps/util/views.py

```diff
@@ -383,6 +384,13 @@ def get_feedback_form_context(request):
 
     context["support_email"] = configuration_helpers.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL)
 
+    #Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)
+    if enterprise_learner_data:
+        enterprise_customer_name = enterprise_learner_data[0]['enterprise_customer']['name']
+        context["tags"]["learner_type"] = "enterprise_learner"
```

Comment:
I don't personally see an issue with it going to datadog; it could be a useful analytic. But someone else should weigh in on that. @brittneyexline, can you point someone (Analytics team?) at this to see?

Note that the Zendesk tags will only get the "enterprise_learner" string value as a tag, which should be OK.